panoramapublic/src/org/labkey/panoramapublic/proteomexchange/SubmissionDataStatus.java

```diff
@@ -109,6 +111,34 @@ public void addMissingRawPath(String filePath, String skylineDoc)
         skyDocs.add(skylineDoc);
     }
 
+    public Map<String, MissingLibrarySourceFiles> getMissingLibFiles()
+    {
+        return _missingLibFiles;
+    }
+
+    public void addMissingLibFile(String lib, String skylineDoc, String[] ssf, String[] idFiles)
```

Comment:
Similar question here - can these be lists instead of arrays? Looks like the calling code is converting a list to an array, which then gets converted back to a list.